### DISCUSSION TITLE
pb_hooks sleep()?

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#3929] [Q&A]
I have a situation where a `cronAdd` handler needs to send out a series of API requests to a rate-limited endpoint. 

I looked for `$os.sleep()` or something similar, but I don't see it. Any ideas?

**Comment by ganigeorgiev** [comment]
There is no `time.Sleep(d)` binding exposed to the JS hooks at the moment (_I'm still not sure if it will be a good idea to have it as it could block the jsvms pool, but I'll consider adding it with the next release as several other users also requested it in the past_).

**Reply by benallfree** [reply]
Ok so my API endpoint is limited to 14 requests per second. It sounds like there is no way to force that, so what would be the alternative? Setting a cron to run every minute and only executing 14 during that minute?
